### ISSUE TITLE
Translation Help (Perpetual)

### ISSUE THREAD

**Issue by JMilamber** [JMilamber/Brass_Amber_BattleTowers#57]
This is an issue to post to if you have translations but don't want to fork the build and add them.
As well as a place for me to ask for translation help when new blocks/items/entities are added so that updates can come out with already correct translations.

**Comment by JMilamber** [comment]
The next 1.16.5 update will have a few renames:

"Golem Chest" -> "Land Golem Chest"
"Stone Chest" -> "Land Chest"

as well as a new item:

"Land Chest Shard"

@Samlegamer renames and the new item are on the Bugfix/Update branch if you want to fork and add, or you can reply here with names and I can add/fix them. Cheers!

**Comment by JMilamber** [comment]
Current 1.16.5 version (1.6.0) renamed several things:

All silver_* items/gear/blocks are now named platinum_*
    Example: silver_sword -> platinum_sword
    
Spawners renamed from bt_spawner to bt_land_spawner

Changes are in the 1.16.5-Bugfix/Update branch 

@Samlegamer

**Comment by JMilamber** [comment]
New Entity needing translation on 1.18.2 branch. Should be in the file already. -added your updates to the 1.16.5 Lang into the branch already. 
@Samlegamer

**Comment by BlackShadow77** [comment]
@JMilamber can you add voices to translate the overscreen sentences when tower is starting to explode?

**Comment by JMilamber** [comment]
@BlackShadow77 By voices I'm guessing you mean lines in the language file that can be translated? Or do you mean actual voice lines? 

Just making sure!
I can definitely add them to the language file. :)

**Comment by BlackShadow77** [comment]
> @BlackShadow77 By voices I'm guessing you mean lines in the language file that can be translated? Or do you mean actual voice lines?

Yes, i mean lines that can be translated, i found nothig about overscreen titles

**Comment by JMilamber** [comment]
@BlackShadow77 @Samlegamer @Kzeroko  1.18.2 Branch language file has new translation lines for the over screen text after defeating the Land Golem. Thanks to BlackShadow for requesting them :)

**Comment by JMilamber** [comment]
> @BlackShadow77 @Samlegamer @Kzeroko 1.18.2 Branch language file has new translation lines for the over screen text after defeating the Land Golem. Thanks to BlackShadow for requesting them :)

**Comment by JMilamber** [comment]
@BlackShadow77 @Samlegamer @Kzeroko Apologies for the back-to-back pings, I had to make a few changes to the en_us file on the 1.18.2 branch to get the Title working correctly again.  So sam you may have to change the file you just submitted slightly, apologies!!

Also added tooltips for the Land chest and Golem Chest

Final changes are on the 1.18.2 branch, confirmed to be actually working.

**Comment by BlackShadow77** [comment]
@JMilamber i updated italian translation.
If you'll need to add some new translation lines can you also put them on translation files keeping them in english? For me can be very helpfull avoid the file comparison step. Also a note on what lines changed can be helpfull in my opinion.

As always thank you for the work and love on this mod, i really apreciate that.

**Comment by JMilamber** [comment]
@BlackShadow77 @Samlegamer @Kzeroko 
lang update- No new Lines

en_us: 
    Lines changed: 
        6-11 (Golem Names) 
        15 & 97 (Platinum Skeleton -> Platinum Knight) 
        101-104 (Item Tooltips)
        111-116 -> (Overscreen titles -  path updated to reflect usage ("name.ba_bt" -> "title.ba_bt")
        
Code: 
    Golem boss bars now use the Entity's name in the lang files as a translatable component instead of being hardcoded.

**Comment by JMilamber** [comment]
@BlackShadow77 @Samlegamer @Kzeroko 
Some back to back additions here lol: 

Also forgot to mention a change to line 107 in my last comment (thanks BlackShadow for catching that) 
"Guardian Has Fallen" -> " Has Fallen"  (Be aware of the single space at the front of this changed line. )

The end of the file for every localization has 6 new lines that correspond to the upcoming advancements being added for 1.18.2.

**Comment by JMilamber** [comment]
@Samlegamer @Kzeroko @BlackShadow77 
Line 103 Changed

Two lines for root BattleTower Advancement were added at 119 and 120

**Comment by JMilamber** [comment]
@Samlegamer @Kzeroko @BlackShadow77 
Lang changed in commit 831dc0692b63a0e88d738ed2593e6fb1c9081662
Go to commit to see line changes.

**Comment by JMilamber** [comment]
Lang Changes in commit https://github.com/BrassAmber-Mods/Brass_Amber_BattleTowers/commit/296fe4ad7d53409c91e61b7c8656eb6449702ebc
Full Change-list in comment https://github.com/BrassAmber-Mods/Brass_Amber_BattleTowers/commit/296fe4ad7d53409c91e61b7c8656eb6449702ebc#commitcomment-80990240

@Samlegamer @Kzeroko @BlackShadow77

**Comment by JMilamber** [comment]
Lang Changes in commits:
https://github.com/BrassAmber-Mods/Brass_Amber_BattleTowers/commit/a5aaa07367fb3975bd341e36f35c855022eae070
https://github.com/BrassAmber-Mods/Brass_Amber_BattleTowers/commit/bef7061eb2061261c7d1db045fa92449de872012
https://github.com/BrassAmber-Mods/Brass_Amber_BattleTowers/commit/e4a6428bd99f305ec045691abe6a58e22c0d582d

@Samlegamer @Kzeroko

@BlackShadow77 (The ones you already updated)

**Comment by JMilamber** [comment]
Lang Changes for 1.20.1 in commits: 

Removed all platinum references, added translation sections for the rest of the chests.
Commits note which lines currently need translation.

All .langs have been updated to have the same ordering as en_us and untranslated lines are in the non en_us lang files.

7c5f827 -> en_us update
8f09d766 -> fr_fr update
6926161e -> it_ch update
83f6bbc1 -> it_it and ru_ru update
2508d669 -> zh_cn update

Full update release is not for a bit still so no timeline o when I would need this for a release. 
Next open beta will be soon.

Been a while ya'll ik xDif you still have time to help that would be awesome. 
@Samlegamer  @Kzeroko @BlackShadow77

**Comment by JMilamber** [comment]
A few more Lang changes in the following commits:

67ba302 -> en_us update
0734551 -> all other langs update

Comments updated across all files to match json validation following the style of @Samlegamer latest translation update
Shards for all chests added on lines 146-149 under "Items" (need translation)
path fixes for resonance stone descriptions.

@Samlegamer  @Kzeroko  @BlackShadow77

**Comment by JMilamber** [comment]
@Samlegamer @Kzeroko @BlackShadow77
Lang changes for 1.20.1 in the following commits:

[6c0e1fe](6c0e1fe1f248071968ca82ee149059d0cbffcd68) -> Added Corrite Blocks 
[eda1426](eda14261a05b643bcc19a46436b589fce7544c2e) -> Removed Excess Overscreen title golem names
[aea6a43](aea6a43d3ffaa4cf846f676c7e82460aea284351) -> Overscreen Titles renamed
[8d6b965](8d6b96558615500d008a6f686a267f0da41cf581) -> Defeated title added for each golem in Overscreen Title
[0877728](08777280db2e43ad5e9cf196545dfe21f377d325) -> Added Core Matter block
[af346d4](af346d48e24a788ca0dda9cc4d27b9bf93256833) -> Fixed corematter to core_matter
[f316f35](f316f3548615bb99ae51a825ddf2aa29195613d6) -> Added Fragment of Obthuuryn entity
[d372a06](d372a0691510925c3082c5c3e54e3dc8d0ffdc41) -> Added Corrite Chiseled Booksehlf and Ladder Blocks

If ya'll are still around sorry its been a while.

**Comment by JMilamber** [comment]
@Samlegamer @Kzeroko @BlackShadow77

More Lang changes for 1.20.1 in the following commits: 
[6f28f13](6f28f13e421192d0772662ec9a05d3025ae79376) -> added other core matter blocks to lang file
[c6e1e57](c6e1e5714183182f132aa369002db35cf921396f) -> added Destruction entities 
[fdf3984](fdf3984c569be7af597746af841df3e9a1481595l) -> Added Core temperature effect and enchantment 

Cheers
